include/big_core_input.h

```diff
@@ -33,7 +33,8 @@ class BigCoreInput : public BigCoreBase
     // RV: Pokud se to nevejde, konkretni data se budou cist az na vyzadani a prectena data se zaroven ulozi do bufferu pro rychlejsi opakovany pristup.
     // RV: Casem treba pridame nejake funkce pro nacteni tile do bufferu, aby se pak mohlo rychle pristupovat pomoci []
     // RV: Neni potreba nastavovat jmeno souboru (ani to neni zadouci).
-    void loadToMemory();
+	// LB: nepredaval jsem jmeno souboru, ale soubor -- ale ted jak je to navrzene, musim predat jmeno souboru, z ceho by to jinak nacital
+    void loadToMemory(const std::string& filename);
```

Comment:
Nevsiml jsem si, ze to nebylo jmeno ale uz ten soubor. Ale stale si asi nerozumime. Metoda openFile ten soubor otevre a ten ifstream si ulozi do promenne. loadToMemory( ) bude bez parametru a pouzije tu promennou.

Cele to musi fungovat tak, ze se na zacatku otevre soubor, prectou se parametry, ale ne data a soubor zustane otevreny. Pokud chci a mam dost pameti, muzi si nacist vsechna data do pameti. Pokud mam malo pameti a data jsou velka, ctu je az v prubehu na vyzadani. V metode closeFile, v destruktoru nebo pri otvirani noveho souboru musim stary soubor zavrit.

Reply:
ok, ale v tom pripade musi byt ten soubor tridni promenna, coz dosud nebyl

Reply:
souhlas